Posh-ACME/DnsPlugins/DomainOffensive.ps1

```diff
@@ -0,0 +1,103 @@
+function Add-DnsTxtDomainOffensive {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory,Position=0)]
+        [string]$Token,
+        [Parameter(Mandatory,Position=1)]
+        [string]$RecordName,
+        [Parameter(Mandatory,Position=2)]
+        [string]$TxtValue
```

Comment:
These need to be positional params 0 and 1. And the `$ExtraParams` one needs to be added back at the end of the param block.

Reply:
Fixed order and added the `$ExtraParams`